src/webdav/Resource.py

```diff
@@ -689,6 +704,7 @@ def manage_DAVget(self):
         # If it doesn't exist, give up.
         return ''
 
+    @zpublish
     @security.protected(webdav_access)
     def listDAVObjects(self):
         return []
```

Comment:
this method had no docstring before, it would become publishable with this change, I don't know if this is intentional.